onnxscript/tests/cli_test.py

```diff
@@ -7,15 +7,10 @@
 
 import click.testing
 
-from onnxscript.__main__ import onnx2script, translate
+from onnxscript.__main__ import translate
```

Comment:
So there’s still translate? Just curious where it was

Reply:
That was incomplete, sorry. Removed it entirely.